### PR TITLE
Allow timezone configuration

### PR DESCRIPTION
**Pull request by curtisgibby**
to fix incorrect "today" highlighting when it's past midnight in UTC, but before midnight in the desired timezone.

### CODE REVIEW

File: src/View/Helper/CalendarHelper.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ class CalendarHelper extends Helper {
 	protected $_defaultConfig = [
 		'monthAsString' => false,
 		'multiLabelSuffix' => ' (Day {0})',
+		'timezone' => 'Etc/UTC',
```
Comment (dereuromark) [inline src/View/Helper/CalendarHelper.php:60]: Can we use null as default, and use configured system default then?

File: src/View/Helper/CalendarHelper.php [hunk 2]
```diff
@@ -183,11 +184,12 @@ public function render() {
 		$month = $this->_View->get('_calendar')['month'];
 
 		$data = $this->dataContainer;
+		$now = new FrozenTime(null, $this->getConfig('timezone', date_default_timezone_get()));
```
Comment (dereuromark) [inline src/View/Helper/CalendarHelper.php:187]: Why cannot we Pass null here? The Code will use this automatically then internally.

**Comment by dereuromark** [comment]
I also added https://github.com/dereuromark/cakephp-calendar/commit/3b6be75830d14c8d9214387c338d7e3a05922672
If you want to check if that is all OK.

**Comment by dereuromark** [comment]
Did what we change impact tests? https://github.com/dereuromark/cakephp-calendar/runs/3306745962?check_suite_focus=true

**Comment by curtisgibby** [comment]
AFAIK, nothing in the tests should have been affected.

**Comment by dereuromark** [comment]
It fails since today for some reason

**Comment by dereuromark** [comment]
Sry, it was the previous PR ( https://github.com/dereuromark/cakephp-calendar/pull/20 )

**Pull request merged**